### PR TITLE
Fix changelog

### PR DESCRIPTION
**Pull request by jack-berg**
Followup to #3236.

### CODE REVIEW

**Review by reyang — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -116,10 +118,6 @@ release.
   ([#3190](https://github.com/open-telemetry/opentelemetry-specification/pull/3190))
 - Expand the declaration of `pool.name`.
   ([#3050](https://github.com/open-telemetry/opentelemetry-specification/pull/3050))
-- Define attributes collected for `cosmosdb` by Cosmos DB SDK
```
Comment (MrAlias) [inline CHANGELOG.md:119]: Should this be removed?

Comment (jack-berg) [reply CHANGELOG.md:119]: Yes, its a duplicate of line 46.

Comment (bertysentry) [reply CHANGELOG.md:119]: My bad! Sorry!

**Review by MrAlias — APPROVED** [review]

**Review by carlosalberto — APPROVED** [review]

**Comment by jack-berg** [comment]
Somehow github let me create a branch against `opentelemetry-specification` using inline editing feature in the UI, but won't let me push updates to it so the branch is stuck in "out-of-date with the base branch". See #3343 instead. Sorry about all this confusion.

**Pull request closed without merging**